docs/data.md

```diff
@@ -0,0 +1,68 @@
+# Data
+
+## View data
+
+Ypu can view and download both environmental an phenotypic variables values from the `View` section of the `Data` menu.
+
+![view-data](img/view-data.png)
+
+Select a variable from the dropdown list at the top left corner of the table to get an overview of the data linked to this variable.
+
+You can refine your search by using the **Date**, **Object** or **Provenance** filters at the top of the table displaying the data associated to the selected variable.
+In order to validate your search, press enter or click anywhere outside the filter insert after having typed the content of your search.
+
+You can dowload the data linked to your search result (and to a single variable) clicking on the <span class="btn btn-primary">Download Search Result</span> button located the top left corner of the data table.
+
+## Add data
+
+### Overview of a dataset import
+
+One can use the web app to manually insert data in Phis, although it is recommended to have this task covered by an automated pipeline, such as a Python scripts which inserts automatically data produced by a sensor to Phis using the `POST /data` web service.
+
+When inserting data in Phis through the webapp, one must provide :
+
+- the name of the **variable(s)** concerned by the data import
+- the source (**provenance**) of this dataset
+- the dataset itself (as a **CSV file** in which fields are separated by semi-colons `;`, and the decimal separator is a point `.`)
```

Comment:
Could you add that you can import numeric, symbolic or date value

Reply:
yes, I will mention it